newdle/client/src/components/NewdleTitle.js

```diff
@@ -41,12 +38,7 @@ export default function NewdleTitle({
                 content={!finished ? 'Answer newdle' : 'This newdle has already finished'}
                 position="bottom center"
                 trigger={
-                  <Button
-                    icon
-                    active={!isSummary}
-                    onClick={() => (isSummary ? history.push(answerURL) : null)}
-                    disabled={finished}
-                  >
+                  <Button icon exact disabled={finished} as={NavLink} to={answerURL}>
```

Comment:
Without the custom onClick logic, does this spam entries in the history whenever you click? (we do not want to add a new history entry when you are already on the page)

Reply:
Yes, it does. Which means that when going back, it will return to the previous button link. I can change it to do the opposite.